lib/aiken/collection/pairs.ak

```diff
@@ -581,21 +581,21 @@ test map_2() {
 /// use aiken/primitive/bytearray
 ///
 /// let compare_un_b_data = fn(l, r) {
-///   bytearray.compare(l |> builtin.un_b_data, r |> builtin.un_b_data)
+///   bytearray.compare(l |> builtin.un_b_datar |> builtin.un_b_data)
```

Comment:
```suggestion
///   bytearray.compare(l |> builtin.un_b_data, r |> builtin.un_b_data)
```

Reply:
Yes, I've fixed it here https://github.com/aiken-lang/stdlib/pull/93/commits/dbda5541f8551546d6d93b51edd6cf8d663dae84